app/insomnia/api-specs.md

```diff
@@ -0,0 +1,85 @@
+    a: Yes. For Insomnia to autodetect that your spec is in GraphQL format, the path must be `/graphql`, the method must be `POST`, the request body must be application/json and must contain a property query with the type string, and the response body must be application/json.
+  - q: Can I create a spec in a collection?
+    a: No, you can import an existing spec into a collection, but you can only create a new API spec in a document.
+---
+
+## What are API specs?
+
+API specifications explain how an API behaves and how it interacts with other APIs. API specs can function like API documentation, but they also explain the values, parameters, and objects in the schema of the API. An API spec helps users know how to make requests to the APIs contained within the spec.
+
+## How does Insomnia parse API specs?
+
+Insomnia parses the following from your API specs and displays these in the Insomnia UI:
+
+| API spec definition | Object in Insomnia |
+| ---- | ---- |
+| `description` | Displays in the Docs tab of a request. |
+| `info` | Displays in the left-sidebar of the spec under INFO. |
+| `servers.url` | Displays in the left-sidebar of the spec under SERVERS. |
+| `paths` | Displays in the left-sidebar of the spec under PATHS. |
+| `requestBodies` | Displays in the left-sidebar of the spec under REQUEST BODIES. |
+| `schemas` | Displays in the left-sidebar of the spec under SCHEMAS. |
+
+## What can I do with API specs in Insomnia?
+
+The following sections provide common API spec use cases you can solve with Insomnia.
+
+### Manage API specs
+
+| Create a new spec directly in the Insomnia editor | In your Insomnia workspace, click **Create** and select **Design document**. ![Create a design document](/assets/images/insomnia/create-a-design-document.png) Your spec editor is in the center pane. ![Spec editor](/assets/images/insomnia/document-spec-editor.png) |
+| Import an existing spec (requirements) | In your workspace, click **Import** and select the source of the import: **File**, **URL**, or **Clipboard**. Click **Scan**. You can review the resources to import, then click **Import**. |
+
+### Linting
+
+| Automatic linting on your specs (new or imported) directly in Insomnia | Navigate to the design document with your API spec. The linting error messages display below the center pane. ![Linting error messages](/assets/images/insomnia/linting-messages.png) |
+| Automate linting on specs using the Inso CLI | Run `inso lint spec` and select the spec from the list that displays. |
+| Use custom linting to ensure your spec adheres to your team or company's standards | To lint locally, create a `.spectral.yaml` file with the custom rules in the same directory as the `oas.yaml` file you want to lint. Then run `inso lint spec ./oas.yaml` from that directory. To apply custom lint rules in the Insomnia UI, add the `.spectral.yaml` file to the root of the collection git repository at the same level as the `.insomnia` folder. |
```

Comment:
Wasn't able to get this one to work locally, haven't tried git sync part yet (was hoping to follow Lucie's instructions once she opens that PR)